common/test/acceptance/fixtures/__init__.py

```diff
@@ -1,11 +1,15 @@
 import os
 
 
+HOSTNAME = os.environ.get('BOK_CHOY_HOSTNAME', 'localhost')
+CMS_PORT = os.environ.get('BOK_CHOY_CMS_PORT', '8031')
```

Comment:
These ports aren't actually configurable yet, because they are hardcoded in the different json files.  Not sure if that can and will be fixed.  If not, should we mention that here in a comment to avoid the false allusion that it is configurable?